### PR TITLE
feat(vitals-alert): updates constant and comments

### PR DESCRIPTION
**Pull request by scefali**
Updates constant and comments

### CODE REVIEW

File: static/app/components/performance/vitalsAlert/constants.tsx [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 };
 
+// these were determined using a Looker query and might change over time
 export const SENTRY_CUSTOMERS: Record<VitalsKey, number> = {
   LCP: 948,
   FCP: 760,
-  appStartCold: 4000, // TODO: Update
-  appStartWarm: 1500, // TODO: Update
+  appStartCold: 2260,
+  appStartWarm: 1900,
```
Comment (evanpurkhiser) [inline static/app/components/performance/vitalsAlert/constants.tsx:16]: can you.. put these in two different PRs lol...

these two things are totally unrelated

Comment (scefali) [reply static/app/components/performance/vitalsAlert/constants.tsx:16]: @evanpurkhiser ugh you’re so annoying

Comment (scefali) [reply static/app/components/performance/vitalsAlert/constants.tsx:16]: Storybook doesn’t even work so I’ll just remove that part

Comment (evanpurkhiser) [reply static/app/components/performance/vitalsAlert/constants.tsx:16]: see there's a good reason right there 😏

Comment (scefali) [reply static/app/components/performance/vitalsAlert/constants.tsx:16]: @evanpurkhiser lol I hate it even more when you're kind of right, removed the storybook file

**Review by udameli — APPROVED** [review]
lol storybook 💀
